### PR TITLE
Get files from elastic by file id [FLYW-1071]

### PR DESCRIPTION
**Pull request by hkethi002**
### Review Checklist

- Tests were added to cover all code changes
- Is there a DB upgrade or fix?
  - If so is it covered by integeration tests and run against dev?
- Would this PR benefit from a test plan? If so, is one provided?
- Documentation was added / updated
- Code and tests follow standards in CONTRIBUTING.md
- Extra care is taken and log messages are created for [Critical Data](https://github.com/flywheel-io/core/wiki/Critical-Data-Flows)

### CODE REVIEW

**Comment by hkethi002** [comment]
Yup

**Comment by nagem** [comment]
Automated testing didn't catch this because we mock ES requests and responses. I'd be interested in other's thoughts on where a wider integration test for this should live.

**Comment by hkethi002** [comment]
I'm planning on adding it to the mc integration, which might make sense because the original change that caused this was in the connector

File: swagger/schemas/definitions/search.json [hunk 1]
```diff
@@ -227,7 +227,7 @@
         "search-ml-input-file": {
             "type": "object",
             "properties": {
-                "_id": {"$ref": "common.json#/definitions/objectid"}
+                "_id": {"type": "string"}
```
Comment (nagem) [inline swagger/schemas/definitions/search.json]: There should be one for a file id, if you wanted to use that.

**Comment by jody-zeitler** [comment]
QA verified with the UI branch on the test2 environment

**Pull request merged**